tox.ini

```diff
@@ -35,11 +34,13 @@ setenv=
     PYTHONDONTWRITEBYTECODE=1
 deps=
     dj22: Django>=2.2,<2.3
-    dj31: Django>=3.1,<3.2
     dj32: Django>=3.2,<3.3
+    dj40: Django>=4.0,<4.1
+    dj40: pytz
```

Comment:
setup.py's `install_requires` should include pytz now as we haven't migrated away from pytz yet. If you don't mind, please add the required changes for migrating away from pytz (it can be in a separate PR).

Reply:
nvm pytz was missing before due to DRF. This is not necessary.